spec/user_stories_spec.rb

```diff
@@ -0,0 +1,82 @@
+# As a Maker
+# So that I can post messages on Chitter as me
+# I want to sign up for Chitter
+feature 'Sign up' do
+  scenario 'display welcome message to new users' do
+    sign_up
+    expect(page).to have_content('Welcome chuck')
+  end
+
+  scenario 'increase user count by one upon sign up' do
+    expect{ sign_up }.to change(User, :count).by(1)
+  end
+
+  scenario 'password does not match password confirmation' do
+      expect { sign_up(password_confirmation: 'wrong') }.not_to change(User, :count)
+      expect(current_path).to eq('/sign_up') # current_path is a helper provided by Capybara
```

Comment:
Metrics/LineLength: Line is too long. [92/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)